src/vs/base/browser/ui/checkbox/checkbox.ts

```diff
@@ -174,3 +180,46 @@ export class Checkbox extends Widget {
 		this.domNode.setAttribute('aria-disabled', String(true));
 	}
 }
+
+export class SimpleCheckbox extends Widget {
+	private checkbox: Checkbox;
+	private styles: ISimpleCheckboxStyles;
+
+	readonly domNode: HTMLElement;
+
+	constructor(private title: string, private isChecked: boolean) {
+		super();
+
+		this.checkbox = new Checkbox({ title: this.title, isChecked: this.isChecked, actionClassName: 'monaco-simple-checkbox' });
```

Comment:
Should it be a subclass?

Reply:
I chose composition to avoid having to use the same style keys or having to modify the old interface.